modules/client-configuration/pages/registration-methods-saltbundle.adoc

```diff
@@ -0,0 +1,32 @@
+[[registration-methods-saltbundle]]
+= Client Registration with Salt Bundle
+
+Since end of 2021, the registration method with the so-called Salt bundle is available.
+The Salt bundle is provided as the [package]``venv-salt-minion`` that consists of Salt and all the packages the version of Salt depends on.
+Using the Salt bundle it is no longer needed that the client provides the required Python version and [package]``python-xml`` package.
+
+If you bootstrap new clients, registration with the Salt bundle is the default method.
+You can switch existing clients to the Salt bundle method.
+If you switch, the [package]``salt-minion`` package and its Python version will stay installed.
+////
```

Comment:
Why is this commenteded?

Reply:
I feel I should change the wording a little bit first.

Reply:
I'll check this now.

Reply:
done and in.